src/main/webapp/data-visualization.js

```diff
@@ -85,7 +86,34 @@ function displayAmChartsMap(
   chart.exporting.menu = new am4core.ExportMenu();
   chart.exporting.menu.align = 'left';
   chart.exporting.menu.verticalAlign = 'top';
-  chart.exporting.filePrefix = 'CensusViz';
+  chart.exporting.filePrefix = title;
+  chart.exporting.menu.items = [
+    {
+      'label': 'Download',
+      'menu': [
+        {
+          'label': 'Image',
+          'menu': [
+            {'type': 'png', 'label': 'PNG'},
+            {'type': 'jpg', 'label': 'JPG'},
+            {'type': 'svg', 'label': 'SVG'},
+            {'type': 'pdf', 'label': 'PDF'},
+          ],
+        }, {
+          'label': 'Data',
```

Comment:
I do not see this in the UI

Reply:
Oh right. This is the default amcharts menu options, but for maps, they actually don't show the data because it's set up as a separate polygon series, not as the chart data. I'll tag @kyracq and ask her if she knows if there's a way to set our data as chart.data and not just polygonSeries.data, but otherwise, I can remove this set of options entirely.

Reply:
Kyra found a way :) Should be done now.